bundles/org.openhab.binding.dsmr/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="dsmr:device_emucs_v1_0">
+		<instruction-set targetVersion="1">
+			<add-channel id="emeter_actual_demand">
+				<type>haywardomnilogic:actualDeliveryBelgiumType</type>
```

Comment:
I don't think the id `haywardomnilogic` will work for `dsmr`. When this is fixed, can either you or @coop-git confirm that the update is working by testing a migration?

Reply:
haha, certainly is not. I should not do this in between other tasks :-/